book/website/reproducible-research/licensing.md

```diff
@@ -10,102 +13,301 @@ No previous knowledge is needed, this chapter explains how important it is to un
+      </div>
+      Licenses can either be Free or Proprietary, with Free Licenses further classified as Copyleft or Permissive.
+    </caption>
+</table>
+
+(rr-licensing-usage-restriction)=
+### What are 'Usage Restricting' Licenses?
+
+Usage restricting licenses seek to affirmatively protect users or others affected by the use of the work by placing  specific restrictions on its use.
+This curtails freedom 0, the freedom to use software 'for any purpose' and prohibiting the use of the software, or other system, for unethical purposes.
+Both 'Ethical source' & 'Responsible AI' Licenses are examples of this approach and seek to place restrictions on the uses to which the licensees can put the software or machine learning systems licensed in this fashion.
+Consequently, these licence by the classical definitions of free and open-source software from the FSF and OSI would not be considered free or open-source licences, they do however generally resemble them in the other three criteria of the definition.
+Their merits versus conventional open-source licenses have been the subject of some debate, and their adoption has thus far been relatively limited.
+
+Even an attribution requirement (the BY in CC-BY) can in some cases be considered a usage restriction.
+For example the Debian project found the [Common Public Attribution License (CPAL)](https://en.wikipedia.org/wiki/Common_Public_Attribution_License) to be incompatible their free-software guidelines for this reason whilst it is approved by the Open Source Initiative.
+In the case of academic works attribution requirements can serve to re-enforce the citation convention with the force of copyright law.
+
+(rr-licensing-choosing)=
+### Where to find open licenses for different types of work
+
+- Code
+    - The [Open Source Initiaitive (OSI)](https://opensource.org/licenses/) maintains a list of [approved Licences](https://opensource.org/licenses/) open-source licences
+    - [Free Software Foundation](https://www.fsf.org/) maintains a [list of GPL-Compatible Free Software Licenses](https://www.gnu.org/licenses/license-list.html#SoftwareLicenses)
+        - [GNU/FSF recomendations](https://www.gnu.org/licenses/license-recommendations.html)
+    - [choosealicense.com](https://choosealicense.com/) provides a tool to guide you through the license choice project.
+    - [Organisation for ethical source](https://ethicalsource.dev/) maintains a list of [ethical source licenses](https://ethicalsource.dev/licenses/)
+- Prose, Images, Audio, Video, Datasets, and similar
+    - [Creative Commons (CC)](https://creativecommons.org/)
+        - [Creative Commons License Chooser](https://creativecommons.org/choose/)
+          ![figure from https://libguides.gwu.edu/opentextbooks/creative_commons](https://s3.amazonaws.com/libapps/accounts/22603/images/CC-License-Chart.png)
+          <!-- alt text needed! -->
```

Comment:
Yes please :smile:

Reply:
Addressed in [45912fe](https://github.com/the-turing-way/the-turing-way/pull/3142/commits/45912fe5d80b75e4aa85524ed621ff07f1d4d33f)